analyses/fastqc-analysis/lsf-script.txt

```diff
@@ -0,0 +1,7 @@
+#BSUB -P project
+#BSUB -J fastqc-analysis
+#BSUB -oo job.out -eo job.err
+#BSUB -n 2
+#BSUB -R "rusage[mem=4GB] span[hosts=1]"
```

Comment:
I think we could consider reducing resource usage by requesting less memory, the job.out file indicates the max memory usage was only 314 MB

Reply:
@sharonfreshour I would leave these resources as they are since we want to ensure that this will work with small and larger fatsq files. We are not requesting a lot of memory anyhow in this particular module overall.